### PR TITLE
HACCP-9: Save page.

### PR DESCRIPTION
**Pull request by beloglazov91**

### CODE REVIEW

File: src/components/hacpp/SaveProgress/saveProgress.html.twig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="hacpp-save-progress">
+  <div class="hacpp-save-progress__message">
+    {{ message|raw }}
+  </div>
+  <div class="hacpp-save-progress__actions">
```
Comment (c-ancia) [inline src/components/hacpp/SaveProgress/saveProgress.html.twig:5]: hacpp typo in the classes too :)

File: src/components/hacpp/SaveProgress/saveProgress.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@import "src/lib.scss";
+
+.hacpp-save-progress {
```
Comment (c-ancia) [inline src/components/hacpp/SaveProgress/saveProgress.scss:3]: so typo here too please

**Review by c-ancia — CHANGES_REQUESTED** [review]

**Comment by beloglazov91** [comment]
@c-ancia Check it again please.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Upcoming release template] - 1971-11-03
 
 ### Added
--
+- Add saveProgress component.
```
Comment (c-ancia) [inline CHANGELOG.md:11]: Can you sync develop please? it should go in a new section and not the template one. Thanks :)

**Review by c-ancia — CHANGES_REQUESTED** [review]

**Pull request merged**